### PR TITLE
[MRG] Add refs to HNN-GUI in tutorials

### PR DESCRIPTION
**Pull request by jasmainak**
cf discussion in #381. This addresses points 2-5

cc @stephanie-r-jones

### CODE REVIEW

File: examples/workflows/plot_simulate_gamma.py [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
-dynamics are driven by Poisson-distributed (noisy) spike trains to layer II/III
-and layer V pyramidal cell populations.
+This example demonstrates how to simulate gamma rhythms via the well
+established pyramidal-interneuron-gamma mechanisms [1]_,
+as detailed in the `HNN GUI gamma tutorial
+<https://jonescompneurolab.github.io/hnn-tutorials/gamma/gamma>`_,
+using HNN-Core.
+
+We recommend you first review the GUI tutorial.
+The workflow below recreates weak gamma rhythm similar to Figures 4 and 5
```
Comment (rythorpe) [inline examples/workflows/plot_simulate_gamma.py:13]:
```suggestion
The workflow below recreates weak gamma rhythms similar to Figures 4 and 5
```

**Review by rythorpe — APPROVED** [review]

**Comment by rythorpe** [comment]
I'm down to merge as soon as CircleCI passes unless we want another pair of eyes to review.

**Comment by jasmainak** [comment]
Go for it :)

**Comment by rythorpe** [comment]
Thanks @jasmainak! We can check those items off the list in #381!

**Pull request merged**